Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_cp.nut

```diff
@@ -546,8 +548,10 @@ void function HardpointThink( HardpointStruct hardpoint )
 			}
 			else if(file.ampingEnabled)//amping or reamping
 			{
-				if(GetHardpointState(hardpoint)<CAPTURE_POINT_STATE_AMPING)
-				SetHardpointState(hardpoint,CAPTURE_POINT_STATE_AMPING)
+				// i have no idea why but putting it CAPTURE_POINT_STATE_AMPING will say 'CONTESTED' on the UI
+				// since whether the point is contested is checked above, putting the hardpoint state to a value of 8 fixes it somehow
+				if(GetHardpointState(hardpoint)<=CAPTURE_POINT_STATE_AMPING)
+					SetHardpointState(hardpoint, 8 ) 
```

Comment:
Question: Why do we have to do this?

Like, it doesn't happen in vanilla, why does it in Northstar?

IMO, we should get to the core of the issue, not just put a band aid over it.

Reply:
It being 3 months ago, I don't have a strong memory but I recall that `sh_consts` does not have the value for amping state I think.

Reply:
Wouldn't that imply some higher level issue in Northstar?

Reply:
It would, I guess. I don't think this warrants a merge block though.

It works.

Reply:
Conversation moved to https://github.com/R2Northstar/NorthstarMods/issues/601